control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3746,7 +3746,7 @@ func (r *HostedControlPlaneReconciler) removeCloudResources(ctx context.Context,
 		}
 
 		if timeElapsed > resourceDeletionTimeout {
-			log.Info("Giving up on resource deletion since there has not been an update before timeout", "timeElapsed", duration.ShortHumanDuration(timeElapsed))
+			log.Info("Giving up on resource deletion after timeout", "timeElapsed", duration.ShortHumanDuration(timeElapsed))
```

Comment:
do we want to include a TODO to account better for "HCCO nor running" use case which is now degraded from 2m to 10m? e.g. `if hcco pod no healthy then return`?